pkg/registry/resolver_test.go

```diff
@@ -698,6 +698,57 @@ func TestResolve(t *testing.T) {
 		},
 		expectedErr:           errors.New(`test/test: workflow/ipi-aws: parameter "NOT_THE_STEP_ENV" is overridden in [test/test] but not declared in any step`),
 		expectedValidationErr: errors.New(`workflow/ipi-aws: parameter "NOT_THE_STEP_ENV" is overridden in [workflow/ipi-aws] but not declared in any step`),
+	}, {
+		name: "Workflow with trap command without grace_period",
+		config: api.MultiStageTestConfiguration{
+			ClusterProfile: api.ClusterProfileGCPCRIO,
+			Workflow:       &awsWorkflow,
+		},
+		workflowMap: WorkflowByName{
+			awsWorkflow: {
+				ClusterProfile: api.ClusterProfileAWS,
+				Pre: []api.TestStep{{
+					LiteralTestStep: &api.LiteralTestStep{
+						As:       "trapper-keeper",
+						From:     "installer",
+						Commands: "trap \"\" SIGINT",
```

Comment:
Nit: You can avoid the need to escape by putting the string into backticks:
```suggestion
						Commands: `trap "" SIGINT`,
```